src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  position: relative;
+  display: flex;
+  align-items: center;
+
+}
+
+.logo {
+  margin-left: 50px;
+  margin-top: 10px;
+}
+
+.nav {
+  position: absolute;
+  right: 50px;
+}
+
+.menu {
+
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.menu li {
+  margin-left: 20px;
+}
+
+.button {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  position: relative;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  font-style: normal;
+  line-height: normal;
+  text-transform: uppercase;
+}
+
+.button:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
```

Comment:
You marked with `plus` in your PR description but I don't see any variable 🤷‍♂️ 
[+] CSS Variable is used for a blue color
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/0a90d491-b2e4-4dae-bf53-df48ee7bb40a)